addons/binding/org.openhab.binding.konnected/README.md

```diff
@@ -0,0 +1,90 @@
+|Humidity|Number:Dimonsionless|Zone Number| This is the channel type for the humidity sensor on a connected DHT22 sensor |
+
+
+
+You will need to configure each channel with the appropriate zone number corresponding to the zone on The Konnected Alarm Panel.
+Then you need to link the corresponding item to the channel.
+
+For the actuator type channels you can also add configuration parameters times, pause and momentary which will be added to the payload that is sent to the Konnected Alarm Panel.
+These parameters will tell the module to pulse the actuator for certain time period.
+A momentary switch actuates a switch for a specified time (in milliseconds) and then reverts it back to the off state. This is commonly used with a relay module to actuate a garage door opener, or with a door bell to send a momentary trigger to sound the door bell.
+A beep/blink switch is like a momentary switch that repeats either a specified number of times or indefinitely.
+This is commonly used with a a piezo buzzer to make a "beep beep" sound when a door is opened, or to make a repeating beep pattern for an alarm or audible warning. It can also be used to blink lights.
+
+DSB1820 temperature probes.
+These are one wire devices which can all be Konnected to the same "Zone" on the Konnected Alarm Panel.
+As part of its transmission  the module will include an unique "address" property of each sensor probe that will be logged to the debug log when received. 
+This needs to be added to the channel if there are multiple probes connected. 
+The default behavior in absence of this configuration will be to simply log the address of the received event.
+A channel should be added for each probe, as indicated above and configured with the appropriate address.  
+
+
+## Full Example
+
+*.items
+
+Contact Front_Door_Sensor " Front Door" {channel="konnected:module:generic:switch"}
+Switch Siren "Siren"   {channel="konnected:module:generic:actuator"}
+
+
+*.sitemap
+
+Switch item=Front_Door_Sensor label="Front Door" icon="door" mappings=[OPEN="Open", CLOSED="Closed"]
+Switch item=Siren label="Alarm Siren" icon="Siren" mappings=[ON="Open", OFF="Closed"]
+            
+*.things
+
+Thing konnected:module:generic "Konnected Module" [ipAddress="http://192.168.30.153:9586", macAddress="1586517"]{
+   Type switch  :   switch  "Front Door"    [channel_zone=1]
+   Type actuator :  actuator    "Siren"    [channel_zone=1 , momentary = 50 , times = 2 , pause = 50]
+   Type humidity    :  humidity "DHT - Humidity"    [channel_zone=1]
+   Type temperature : temperature   "DHT Temperature"    [channel_zone=1, tempsensorType = true, pollinterval = 1]
+   Type temperature : temperature   "DS18B20 Temperature"    [channel_zone=1, tempsensorType = false, pollinterval = 1, ds18b20_address = XX:XX:XX:XX:XX:XX:XX]
```

Comment:
pretty sure it should be `ds18b20_address = "XX:XX:XX:XX:XX:XX:XX"`

Reply:
correct made the change